themes/bootstrap3/templates/RecordDriver/DefaultRecord/list-entry.phtml

```diff
@@ -185,7 +188,10 @@
       </div>
 
       <div class="result-links hidden-print">
-        <?=$this->icon('user-list-entry-edit') ?> <a href="<?=$this->url('myresearch-edit')?>?id=<?=urlencode($id)?>&amp;source=<?=urlencode($source)?><?php if (null !== $list_id):?>&amp;list_id=<?=urlencode($list_id)?><?php endif; ?>" class="edit tool"><?=$this->transEsc('Edit')?></a><br/>
+        <a href="<?=$this->url('myresearch-edit')?>?id=<?=urlencode($id)?>&amp;source=<?=urlencode($source)?><?php if (null !== $list_id):?>&amp;list_id=<?=urlencode($list_id)?><?php endif; ?>" class="edit tool icon-link">
+          <?=$this->icon('user-list-entry-edit', 'icon-link__icon') ?>
+          <span class="icon-link__label"><?=$this->transEsc('Edit')?></span>
```

Comment:
This link looks good, but if you click on it, the destination page has a "delete" link that needs to be fixed up:
![image](https://user-images.githubusercontent.com/309069/197259091-1860740e-ac6c-4b69-9212-8b3fbe76151f.png)